source/frontend/.eslintrc.json

```diff
@@ -24,7 +24,8 @@
     ],
     "react/display-name": "off",
     "react/prop-types": "off",
-    "no-prototype-builtins": "off"
+    "no-prototype-builtins": "off",
+    "no-debugger": "off"
```

Comment:
**nit:**  I'm all for improving our linting rules but this PR seemed focused on backend changes. Fine for now, but we should try to keep changes relevant to their stories